cob_hardware_config/cob4-5/config/head_driver.yaml

```diff
@@ -0,0 +1,46 @@
+bus:
```

Comment:
theoretically we should remove these lines because we are already setting  the values on the launch file https://github.com/ipa320/cob_robots/blob/feature/enable_head/cob_bringup/drivers/canopen_402.launch#L21 , there parameters are not being used/overwritten , see https://github.com/ipa320/cob4/issues/349#issuecomment-296593818